corehq/apps/export/templates/export/partial/new_customize_export_templates.html

```diff
@@ -14,48 +14,48 @@
-                        css: {active: table.showAdvanced}">
-                        <span data-bind="visible: !table.showAdvanced()">
-                        {% if export_instance.type == 'form' %}
-                            {% trans "Show Deleted Questions and Advanced Properties" %}
-                        {% else %}
-                            {% trans "Show Deleted Properties and Advanced Properties" %}
-                        {% endif %}
-                        </span>
-                        <span data-bind="visible: table.showAdvanced">
-                        {% if export_instance.type == 'form' %}
-                            {% trans "Hide Deleted Questions and Advanced Properties" %}
-                        {% else %}
-                            {% trans "Hide Deleted Properties and Advanced Properties" %}
-                        {% endif %}
-                        </span>
-                    </button>
-                </div>
+        <div class="col-sm-12" data-bind="visibleFade: table.selected">
+            <div class="form-group pull-right">
+                <button type="button" class="btn btn-default" data-bind="
+                    click: table.toggleShowAdvanced,
+                    css: {active: table.showAdvanced}">
+                    <span data-bind="visible: !table.showAdvanced()">
+                    {% if export_instance.type == 'form' %}
+                        {% trans "Show Deleted Questions and Advanced Properties" %}
+                    {% else %}
+                        {% trans "Show Deleted Properties and Advanced Properties" %}
+                    {% endif %}
+                    </span>
+                    <span data-bind="visible: table.showAdvanced">
+                    {% if export_instance.type == 'form' %}
+                        {% trans "Hide Deleted Questions and Advanced Properties" %}
+                    {% else %}
+                        {% trans "Hide Deleted Properties and Advanced Properties" %}
+                    {% endif %}
+                    </span>
+                </button>
             </div>
-            <div class="control-group">
-                <label class="control-label">{% trans "Sheet Name" %}</label>
-                <div class="controls">
-                    <input type="text" data-bind="value: table.label" />
+            <div class="form-group">
+                <label class="col-sm-2 control-label">{% trans "Sheet Name" %}</label>
+                <div class="col-sm-4">
+                    <input type="text" class="form-control" data-bind="value: table.label" />
                 </div>
             </div>
-            <div class="control-group">
-                <label class="control-label">
+            <div class="form-group">
+                <label class="col-sm-2 control-label">
                     <strong>{% trans "Choose the fields you want to export." %}</strong><br />
+                    <span style="font-weight: normal">
```

Comment:
this was the previous implementation, happy to change if you think it's better. it doesn't look horrible and we have extra space in that margin so i could go either way